### PR TITLE
BD-2561: Personalized Paths beta [MERGE 11/6]

### PR DESCRIPTION
**Pull request by bre-fitzgerald**
# Pull Request/Issue Resolution

#### Description of Change:
Adds new doc for Personalized Paths in Experiment Steps

Closes #**BD-2561**

#### Is this change associated with a Braze feature/product release?
- [x] Yes (**November 7**)
- [ ] No

### CODE REVIEW

File: _docs/_user_guide/engagement_tools/canvas/canvas_components/experiment_step/personalized_paths.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+Add an [Experiment Path]({{site.baseurl}}/user_guide/engagement_tools/canvas/canvas_components/experiment_step/) to your Canvas, then turn on **Personalized Paths**.
+
+![][1]{: style="max-width:75%;" }
+
+### Step 2: Configure Personalized Paths settings
+
+Specify the conversion event that should determine the winner. If there are no conversion events available, return to the first step of Canvas setup and [assign conversion events]({{site.baseurl}}/user_guide/engagement_tools/canvas/create_a_canvas/create_a_canvas/#choose-conversion-events). Note that if you determine the winner with opens and clicks, only the first message in the path that generates opens or clicks will contribute to determining the winner.  
+
+Then set the **Experiment Window**. The **Experiment Window** determines how long users will be sent down all paths before choosing a single winning path for all subsequent users. The window begins when the first user enters the step.
```
Comment (borisrev) [inline _docs/_user_guide/engagement_tools/canvas/canvas_components/experiment_step/personalized_paths.md:35]: before choosing "the best paths for each user in the delay group."

File: _docs/_user_guide/engagement_tools/canvas/canvas_components/experiment_step/personalized_paths.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+![][4]
+
+### Step 4: Launch the Canvas
+
+Finish setting up your Canvas as needed, then launch it. When the first user has entered the experiment, you can check the Canvas to see analytics as they come in.
+
+![][5]{: style="max-width:75%;" }
+
+When the experiment window passes and the experiment is complete, Braze will send users in the delay group to their respective paths.
```
Comment (borisrev) [inline _docs/_user_guide/engagement_tools/canvas/canvas_components/experiment_step/personalized_paths.md:57]: optional: "to their respective paths with the highest personalized likelihood of conversion."

**Review by borisrev — CHANGES_REQUESTED** [review]
Minor tweaks otherwise lgtm

File: _docs/_user_guide/engagement_tools/canvas/canvas_components/experiment_step/personalized_paths.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+nav_title: Personalized Paths 
+article_title: Personalized Paths in Experiment Paths 
+page_type: reference
+description: "Personalized Paths is similar to Personalized Variant in campaigns and lets you automate your A/B tests."
+tool: Canvas
+---
+
+# Personalized Paths in Experiment Paths
+
+> Personalized Paths is similar to [Personalized Variant]({{site.baseurl}}/user_guide/engagement_tools/testing/multivariant_testing/optimizations/#personalized-variant) in campaigns and lets you automate A/B tests in your Canvas.
```
Comment (borisrev) [inline _docs/_user_guide/engagement_tools/canvas/canvas_components/experiment_step/personalized_paths.md:11]: "in campaigns and lets you personalize entire journeys for individual users based on conversion likelihood."

Comment (bre-fitzgerald) [inline _docs/_user_guide/engagement_tools/canvas/canvas_components/experiment_step/personalized_paths.md:11]:
```suggestion
> Personalized Paths is similar to [Personalized Variant]({{site.baseurl}}/user_guide/engagement_tools/testing/multivariant_testing/optimizations/#personalized-variant) in campaigns and lets you personalize entire Canvas journeys for individual users based on conversion likelihood.
```

File: _docs/_user_guide/engagement_tools/canvas/canvas_components/experiment_step/personalized_paths.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+Add an [Experiment Path]({{site.baseurl}}/user_guide/engagement_tools/canvas/canvas_components/experiment_step/) to your Canvas, then turn on **Personalized Paths**.
+
+![][1]{: style="max-width:75%;" }
+
+### Step 2: Configure Personalized Paths settings
+
+Specify the conversion event that should determine the winner. If there are no conversion events available, return to the first step of Canvas setup and [assign conversion events]({{site.baseurl}}/user_guide/engagement_tools/canvas/create_a_canvas/create_a_canvas/#choose-conversion-events). Note that if you determine the winner with opens and clicks, only the first message in the path that generates opens or clicks will contribute to determining the winner.  
+
+Then set the **Experiment Window**. The **Experiment Window** determines how long users will be sent down all paths before choosing a single winning path for all subsequent users. The window begins when the first user enters the step.
```
Comment (bre-fitzgerald) [inline _docs/_user_guide/engagement_tools/canvas/canvas_components/experiment_step/personalized_paths.md:35]:
```suggestion
Then set the **Experiment Window**. The **Experiment Window** determines how long users will be sent down all paths before choosing the best path for each user in the delay group. The window begins when the first user enters the step.
```

File: _docs/_user_guide/engagement_tools/canvas/canvas_components/experiment_step/personalized_paths.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+
+![][4]
+
+### Step 4: Launch the Canvas
+
+Finish setting up your Canvas as needed, then launch it. When the first user has entered the experiment, you can check the Canvas to see analytics as they come in.
+
+![][5]{: style="max-width:75%;" }
+
+When the experiment window passes and the experiment is complete, Braze will send users in the delay group to their respective paths.
```
Comment (bre-fitzgerald) [inline _docs/_user_guide/engagement_tools/canvas/canvas_components/experiment_step/personalized_paths.md:57]:
```suggestion
When the experiment window passes and the experiment is complete, Braze will send users in the delay group to their respective paths with the highest personalized likelihood of conversion.
```

**Review by borisrev — APPROVED** [review]

File: _docs/_user_guide/engagement_tools/canvas/canvas_components/experiment_step/personalized_paths.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
+---
+nav_title: Personalized Paths 
+article_title: Personalized Paths in Experiment Paths 
+page_type: reference
+description: "Personalized Paths is similar to Personalized Variant in campaigns and lets you automate your A/B tests."
```
Comment (bre-fitzgerald) [inline _docs/_user_guide/engagement_tools/canvas/canvas_components/experiment_step/personalized_paths.md:5]:
```suggestion
description: "Personalized Paths is similar to Personalized Variant in campaigns and lets you personalize entire Canvas journeys based on each person's conversion likelihood."
```

**Pull request merged**